crates/rnote-engine/src/lib.rs

```diff
@@ -18,6 +18,7 @@ pub mod ext;
 pub mod fileformats;
 pub mod pens;
 pub mod render;
+pub mod selectioncollision;
```

Comment:
Re-Export the enum below to avoid having to import `selectioncollision::SelectionCollision`